x-pack/plugins/security_solution/server/lib/risk_engine/tasks/risk_scoring_task.ts

```diff
@@ -265,6 +267,9 @@ export const runTask = async ({
 
     telemetry.reportEvent(RISK_SCORE_EXECUTION_SUCCESS_EVENT.eventType, telemetryEvent);
 
+    if (isCancelled()) {
+      log('task was cancelled');
```

Comment:
@nkhristinin we could potentially add a `wasCancelled` boolean to _both_ success and error events; I think that would give us a good indication of when things are working but slow (success + wasCancelled), and when the slowness is causing/due to other errors (error + wasCancelled)

Reply:
Additionally: with this new task cancellation logic, `executionDurationExceededInterval` is never going to be true because that cancellation is going to stop the task on the next loop; I can't imagine a situation where that last loop takes > 50 minutes for the interval to be exceeded.

Reply:
I like the idea of having a separate event. I'll whip that up in a separate PR 👍